### ISSUE TITLE
Parser can't cope with argument of type "type"

### ISSUE THREAD

**Issue by jcb91** [terrencepreilly/darglint#77]
Using the google-style parser, specifying an argument to have type `type` (that is, the argument passed should be a class, as opposed to an instance of a class) appears to confuse the parser, causing it to raise false-positive `DAR101` and/or `DAR003`.

Best demonstrated in code, I expect:

```python
def my_func_darglint_satisfied(arg_a, arg_b):
    """Do nothing. Darglint doesn't complain.

    Args:
        arg_a (object): The first argument.
        arg_b (int): The second argument.
    """


def my_func_darglint_complains_1(arg_a, arg_b, arg_c):
    """Do nothing. Darglint complains that arg_a, arg_b and arg_c are missing.

    Args:
        arg_a (type): The first argument.
        arg_b (int): The second argument.
        arg_c (int): The third argument.
    """
    # For this docstring, darglint sees all args as missing:
    #
    #     tmp.py:my_func_darglint_complains_1:10: DAR101: - arg_a
    #     tmp.py:my_func_darglint_complains_1:10: DAR101: - arg_c
    #     tmp.py:my_func_darglint_complains_1:10: DAR101: - arg_b


def my_func_darglint_complains_2(arg_a, arg_b, arg_c):
    """Do nothing. Darglint will complain.

    Args:
        arg_a (int): The first argument. Darglint seems to find this ok.
        arg_b (type): The second argument. Darglint complains here.
        arg_c (int): The third argument. Darglint doesn't complain about this.
    """
    # For this docstring, darglint sees arg_b as missing, and complains about indentation:
    #
    #     tmp.py:my_func_darglint_complains_2:28: DAR101: - arg_b
    #     tmp.py:my_func_darglint_complains_2:29: DAR003: ~


def my_func_darglint_complains_3(arg_a, arg_b, arg_c):
    """Do nothing. Darglint will complain.

    Args:
        arg_a (int): The first argument. Darglint seems to find this ok.
        arg_b (int): The second argument. Darglint seems to find this ok.
        arg_c (type): The third argument. Darglint complains here.
    """
    # For this docstring, darglint sees arg_c as missing, and complains about indentation:
    #
    #     tmp.py:my_func_darglint_complains_3:42: DAR101: - arg_c
    #     tmp.py:my_func_darglint_complains_3:44: DAR003: ~<

```

swapping `type` for `object` removes the errors, but isn't really what should go in the docstring.

**Comment by terrencepreilly** [comment]
Ah, yes.  It looks like used some terminal tokens instead of the identifier token in the grammar ([here](https://github.com/terrencepreilly/darglint/blob/db76ec90a47f3bf7e8b85bdecca115ea75d4f7f4/darglint/parse/grammars/google_types.bnf#L26).)  It should be an easy fix.  I'll try to get to it this weekend.

**Comment by terrencepreilly** [comment]
Should be resolved in 62e58c9f32c207d2bd149e700710e4e31a0be014, which will get included in v1.2.2.
